packages/grid_client/src/modules/calculator.ts

```diff
@@ -59,10 +59,13 @@ class Calculator {
     let balance = 0;
     const pricing = await this.pricing(options);
 
+    // certified node cotsts 25% more than DIY node
+    const certified = options.certified ? 1.25 : 1;
+
     // discount for Dedicated Nodes
     const discount = pricing.dedicatedDiscount;
-    let dedicatedPrice = pricing.musd_month - pricing.musd_month * (+discount / 100);
-    let sharedPrice = pricing.musd_month;
+    let dedicatedPrice = pricing.musd_month - pricing.musd_month * (+discount / 100) * certified;
```

Comment:
I think we should apply the certified factor on the total price not only on the discount